README.md

```diff
@@ -59,72 +67,31 @@ You will have to copy driver to directory in `PATH` so `docker-machine` would be
-`--otc-password`            | `OS_PASSWORD`             |                                       | OpenTelekomCloud Password
-`--otc-private-key-file`    | `OS_PRIVATE_KEY_FILE`     |                                       | Private key file to use for SSH (absolute path)
-`--otc-project-id`          | `OS_PROJECT_ID`           |                                       | OpenTelekomCloud Project ID
-`--otc-project-name`        | `OS_PROJECT_NAME`         |                                       | OpenTelekomCloud Project name
-`--otc-region`              | `REGION`                  | eu-de                                 | Region name
-`--otc-root-volume-size`    | `ROOT_VOLUME_SIZE`        | 40                                    | Set volume size of root partition (in GB)
-`--otc-root-volume-type`    | `ROOT_VOLUME_TYPE`        | SATA                                  | Set volume type of root partition (one of `SATA`, `SAS`, `SSD`)
-`--otc-sec-groups`          | `OS_SECURITY_GROUP`       |                                       | Existing security groups to use, separated by comma
-`--otc-skip-default-sg`     |                           |                                       | Don't create default security group
-`--otc-skip-ip`             |                           |                                       | If set, elastic IP won't be created, machine IP will be set to instance local IP
-`--otc-ssh-port`            | `OS_SSH_PORT`             | 22                                    | Machine SSH port
-`--otc-ssh-user`            | `SSH_USER`                | ubuntu                                | SSH user
-`--otc-subnet-id`           | `SUBNET_ID`               |                                       | Subnet id the machine will be connected on
-`--otc-subnet-name`         | `SUBNET_NAME`             | subnet-docker-machine                 | Subnet name the machine will be connected on
-`--otc-token`               | `OS_TOKEN`                |                                       | Authorization token
-`--otc-tenant-id`           | `TENANT_ID`               |                                       | Project ID. DEPRECATED: use `-otc-project-id` instead
-`--otc-user-data-file`      | `OS_USER_DATA_FILE`       |                                       | File containing an userdata script
-`--otc-username`            | `OS_USERNAME`             |                                       | OpenTelekomCloud username
-`--otc-vpc-id`              | `VPC_ID`                  |                                       | VPC id the machine will be connected on
-`--otc-vpc-name`            | `OS_VPC_NAME`             | vpc-docker-machine                    | VPC name the machine will be connected on
+
+For versions `v0.3.x` see [supported-options](docs/supported-options-v0.3.x.md)
+For versions `v0.2.x` see [supported-options](docs/supported-options-v0.2.x.md)
```

Comment:
Those two lines are concatenated in rendered markdown

Reply:
done